packages/typescript/BUILD.bazel

```diff
@@ -47,16 +47,19 @@ npm_package(
     name = "npm_package",
     srcs = [
         "defs.bzl",
+        "index.bzl",
         "package.bzl",
         "package.json",
         "version.bzl",
+        "//devserver:npm_package_assets",
         "//internal:npm_package_assets",
         "//ts_auto_deps:npm_package_assets",
     ],
     packages = [
         "@build_bazel_rules_typescript//:npm_bazel_typescript_package",
     ],
     replacements = {
+        "(#|\/\/)\\s+BEGIN-DEV-ONLY[\\w\W]+?(#|\/\/)\\s+END-DEV-ONLY": "",
```

Comment:
we can just use the built-in `# BEGIN-INTERNAL` fences again now that we aren't in a repo that copybara excludes things from

Reply:
ahhh.. but I think there is still some `BEGIN-DEV` fencing in the files vendored in from @build_bazel_rules_typescript//:npm_bazel_typescript_package?

Reply:
right...